classes/utils/Vector.py

```diff
@@ -0,0 +1,90 @@
+from math import sqrt, floor
+
+
+class Vector2:
```

Comment:
I'd use Vector2D here

Reply:
Done, I've also changed Vector3 to Vector3D